internal/subscription/core.go

```diff
@@ -204,6 +219,8 @@ func (c *Core) Get(ctx context.Context, key string) (*Model, error) {
 	}
 	prefix := common.GetBasePrefix() + Prefix + projectID + "/" + subscriptionName
 
+	logger.Ctx(ctx).Infow("fetching subscription", "key", prefix)
```

Comment:
was this a test logger to be removed later?

Reply:
was added as test log, but thought of keeping it for debugging